tests/loader_regression_tests.cpp

```diff
@@ -57,6 +57,7 @@ class SortedPhysicalDevices : public RegressionTests {};
 class CreateDevice : public RegressionTests {};
 class EnumeratePhysicalDeviceGroups : public RegressionTests {};
 class WrapObjects : public RegressionTests {};
+class VerifyGetProcAddr : public RegressionTests {};
```

Comment:
There is a `loader_get_proc_addr_tests.cpp`. This test might be more at home in that file.

Reply:
I didn't see it.  Thanks!